components/ui/DeferredSpinner.tsx

```diff
@@ -79,6 +81,11 @@ export default class DeferredSpinner extends React.PureComponent<Props, State> {
         )
       );
     }
-    return this.props.children || null;
+    return (
+      this.props.children ||
+      (this.props.placeholder ? (
+        <i className={classNames('deferred-spinner-placeholder', this.props.className)} />
+      ) : null)
```

Comment:
What's the use case for this?

Reply:
Use case is to stop using this class alone and in different ways in the SC codebase. So once this version is released I'll replace all the class usage with the `placeholder={true}` prop